lib/reporter/index.js

```diff
@@ -27,7 +27,7 @@ class Reporter extends SimplifiedReporter {
 
     this.suiteRetries = suiteRetries;
     this.suiteName = addOpts.suiteName;
-    this.testResults = new TestResults(tests, addOpts);
+    this.testResults = new TestResults(tests, addOpts, settings.desiredCapabilities);
```

Comment:
Selenium 4.5 has dropped the `desiredCapabilities` so it's no longer required to defined them in nightwatch. What happens if users only define `capabilities` instead?

Reply:
I will make a change to check for both desiredCapabilities and capabilities.